### DISCUSSION TITLE
Change Localization or language

### DISCUSSION THREAD

**Discussion by darksider90** [slipx06/sunsynk-power-flow-card#223] [Q&A] [answered]
Hello and thank you for this wonderfull Job !

I would like to change the name of the text information like "daily load" into french.  I see the folder for the localization, but i dont know how use it into home assistant or change manualy the text.
 Someone could you help me ?

Thanks

**Comment by slipx06** [comment]
Hi. The card has been translated to French. You just need to set your HA language under your user profile

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/0df7c6e1-1486-466c-a7c1-fc203f94c041)

```
    "common": {
        "description": "Une carte Home Assistant pour émuler les flux d'énergie affichés à l'écran des onduleurs Sunsynk ",
        "version": "Version",
        "runtime_to": "DÉCHARGÉE À",
        "battery_floating": "BATTERIE FLOTTANTE",
        "charge": "CHARGE",
        "to": "À",
        "daily_load": "CHARGE (JOUR)",
```

**Comment by darksider90** [comment]
Oh thanks you. I'm Already in French but the text is in english.

After change localizaton to english and rollback to french. It's OK :)

Thank you

**Comment by slipx06** [comment, marked as answer]
This issue was fixed in v4.22
